docs/docs/configuration/service.mdx

```diff
@@ -18,6 +18,21 @@ service Example {
 
 ## Options
 
+### enable_parent_task_queue_for_child_workflows
+
+`bool`
+
+Use parent workflow task queue for child workflows.
+
+```protobuf
+service Example {
+  option (temporal.v1.service) = {
+    task_queue: 'example-v1'
+    enable_parent_task_queue_for_child_workflows: true
```

Comment:
Wondering if `keep_parent_task_queue_for_child_workflows` would be more "self-documenting" ?